### PR TITLE
[REFACTOR] Update the example for fit after load

### PR DESCRIPTION
**Pull request by csouchet**
Closes https://github.com/process-analytics/bpmn-visualization-js/issues/738

- Add fit center button
- Replace mxgraph function to fit, by fit function of the library

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/738-update_example_for_fit_after_load/examples/index.html

### CODE REVIEW

File: examples/diagram-fit-variants/index.html [hunk 1]
```diff
@@ -144,42 +150,22 @@ <h2>Various fit variants</h2>
...
+            bpmnVisualization.fit({type: bpmnvisu.FitType.HorizontalVertical});
+        };
+        document.getElementById('btn-fit-center').onclick = function() {
+            bpmnVisualization.fit({type: bpmnvisu.FitType.Center});
         };
-
-        function performFit(type) {
-            if (type) {
-                fitType = type;
-            }
```
Comment (tbouffard) [inline examples/diagram-fit-variants/index.html:165]: :warning: this breaks the margin management.
Why not keeping this function and update the implementation to use the bpmn-viusalization API? 
this implies less changes and make the margin changes work

File: examples/diagram-fit-variants/index.html [hunk 2]
```diff
@@ -125,7 +128,10 @@ <h2>Various fit variants</h2>
     </section>
 
     <!-- load bpmn-visualization -->
-    <script src="https://unpkg.com/bpmn-visualization@0.6.0/dist/bpmn-visualization.js"></script>
+    <!-- To uncomment on 0.7.0 release -->
+<!--    <script src="https://unpkg.com/bpmn-visualization@0.7.0/dist/bpmn-visualization.js"></script>-->
+    <script src="../../demo/0.7.0/bpmn-visualization.js"></script>
```
Comment (tbouffard) [inline examples/diagram-fit-variants/index.html:133]: :warning: clash with #91

**Review by tbouffard — CHANGES_REQUESTED** [review]

**Review by aibcmars — APPROVED** [review]
LGTM

**Review by aibcmars — CHANGES_REQUESTED** [review]
ahh yes, margin does not work

**Review by aibcmars — APPROVED** [review]

File: examples/diagram-fit-variants/index.html (hunk 1 above)
Comment (tbouffard) [reply examples/diagram-fit-variants/index.html]: now fixed

File: examples/diagram-fit-variants/index.html (hunk 2 above)
Comment (tbouffard) [reply examples/diagram-fit-variants/index.html:133]: we will manage this at release time

**Review by tbouffard — APPROVED** [review]

**Pull request merged**